src/components/BreweryList.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { render } from 'react-testing-library'
+
+import BreweryList from './BreweryList'
+
+describe('BreweryList', () => {
+  it('renders without crashing', () => {
+    const backToSearch = () => {}
+    const { container } = render(
+      <BreweryList backToSearch={backToSearch}>
```

Comment:
If you wanted to mock a function as in `backToSearch` you should use jest's mock function:
` const backToSearch = jest.fn()`